deploy/addons/assets.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import "embed"
+
+var (
```

Comment:
We could instead have one BIG embed.FS for all the files. I feel however this could be confusing as to what addons need what files, or if all deploy/addons files are included, we could include unnecessary files (like the old system).